spec/controllers/application_controller_spec.rb

```diff
@@ -93,21 +84,17 @@
   context "#find_checked_items" do
     it "returns empty array when button is pressed from summary screen with params as symbol" do
       controller.instance_variable_set(:@_params, :id => "1")
-      result = controller.send(:find_checked_items)
-      result.should eq([])
+      expect(controller.send(:find_checked_items)).to eq([])
     end
 
     it "returns empty array when button is pressed from summary screen with params as string" do
       controller.instance_variable_set(:@_params, "id" => "1")
-      result = controller.send(:find_checked_items)
-      result.should eq([])
+      expect(controller.send(:find_checked_items)).to eq([])
     end
 
     it "returns list of items selected from list view" do
       controller.instance_variable_set(:@_params, :miq_grid_checks => "1, 2, 3, 4")
-      result = controller.send(:find_checked_items)
-      result.count eq(4)
-      result.should eq([1, 2, 3, 4])
+      expect(controller.send(:find_checked_items)).to eq([1, 2, 3, 4])
```

Comment:
I don't know if `find_checked_items` sorts, could this hardcoded order be a sporadic problem?